modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -59,25 +59,25 @@ For more information about which permissions to use, see xref:api-manager::org-c
 * Obtain the Cluster Administrator role to access the Kubernetes cluster (only required for installation process).
 * If you have configured an external identity provider for your organization, create a user in Anypoint Platform with Exchange Contributors and Manage APIs Configuration roles. Anypoint Service Mesh does not support users residing on an external identity provider for tasks such as creating and discovering APIs.
 
-== Ports, IPs, and Hostnames Whitelist Requirements
+== Ports, IPs, and Hostnames allow list Requirements
 
-In your network, you might need to whitelist the hostnames and ports of external resources to allow Anypoint Service Mesh to communicate with the MuleSoft-managed online Anypoint Platform APIs and services.
+In your network, you might need to add to the allow list the hostnames and ports of external resources to enable Anypoint Service Mesh to communicate with the MuleSoft-managed online Anypoint Platform APIs and services.
 
-[%header%autowidth.spread,cols="3+a"]
+[%header%autowidth.spread]
 |===
-|Plane |Host |Port
-|*US*|*anypoint.mulesoft.com* | 443
-|*US*|*analytics-ingest.anypoint.mulesoft.com* |  443
-|*US*|**.dkr.ecr.us-east-1.amazonaws.com* |  443
-|*US*|*exchange-files.anypoint.mulesoft.com* |  443
-|*US*|*exchange2-asset-manager-kprod.s3.amazonaws.com* |  443
-|*EU*|*eu1.anypoint.mulesoft.com* | 443
-|*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com* |  443
-|*EU*|**.dkr.ecr.eu-central-1.amazonaws.com* |  443
-|*EU*|*exchange-files.eu1.anypoint.mulesoft.com* |  443
-|*EU*|*exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com* |  443
-|*ALL*|*get.helm.sh* | 443
-|*ALL*|*storage.googleapis.com* | 443
+|Plane |Host |Port| Explanation
+|*US*|*anypoint.mulesoft.com* | 443| Required to connect with the control plane
+|*US*|*analytics-ingest.anypoint.mulesoft.com* |  443| Required to send analytics to the control plane
+|*US*|**.dkr.ecr.us-east-1.amazonaws.com* |  443| Required to download the docker images that make up the Anypoint Service Mesh product
+|*US*|*exchange-files.anypoint.mulesoft.com* |  443| Required to download the docker images that make up the Anypoint Service Mesh product
+|*US*|*exchange2-asset-manager-kprod.s3.amazonaws.com* |  443| Required to download policies
+|*EU*|*eu1.anypoint.mulesoft.com* | 443| Anypoint
+|*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com* |  443| Required to send analytics to the control plane
+|*EU*|**.dkr.ecr.eu-central-1.amazonaws.com* |  443| Required to download the docker images that make up the Anypoint Service Mesh product
+|*EU*|*exchange-files.eu1.anypoint.mulesoft.com* |  443| Required to download policies
+|*EU*|*exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com* |  443| Required to download policies
+|*ALL*|*get.helm.sh* | 443| Required to retrieve and install Helm on the Kubernetes clusters.
```

Comment:
This seems wrong, we don't install Helm into the cluster, only into the users's machine